examples/shell/shell_common/cmd_send.cpp

```diff
@@ -47,7 +47,7 @@ class SendArguments
     {
         mProtocolId   = 0x0002;
         mMessageType  = 1;
-        mLastSendTime = 0;
+        mLastSendTime = System::Clock::Zero;
```

Comment:
should the constant be named kZero ?

Reply:
Yes. I'll do that in a separate PR since it'll touch many files.